src/Map.jsx

```diff
@@ -246,13 +297,14 @@ class Map extends Component {
         this.state.showStops
           ? getStopMarkersLayer(
             route,
-            marker => this.getStopInfo(route, marker.object.position), this.state.selectedStops,
+            marker => this.getStopInfo(route, marker.object.position),
+            selectedStops[route.rid],
           )
           : null,
         getRoutesLayer(geojson),
+        ...this.getSubRoutes(),
```

Comment:
@EddyIonescu 
do not merge yet. Trying to figure out how to handle multiple subroutes
Unless multiple subroutes should be a seperate feature?